tests/reference/ast-prefix_string_01-cf221fd.stderr

```diff
@@ -0,0 +1,5 @@
+syntax error: Token '"Hello"' (of type 'string') is unexpected here
+ --> tests/errors/prefix_string_01.py:3:8
+  |
+3 |  print "Hello", "World" 
+  |        ^^^^^^^ 
```

Comment:
In main we get:
```console
$ lpython b.py 
Internal Compiler Error: Unhandled exception
Traceback (most recent call last):
  File "/Users/ondrej/repos/lpython/src/bin/lpython.cpp", line 1872
    err = compile_python_to_object_file(arg_file, tmp_o, runtime_library_dir,
  File "/Users/ondrej/repos/lpython/src/bin/lpython.cpp", line 771
    LCompilers::Result<LCompilers::LPython::AST::ast_t*> r = parse_python_file(
  File "/Users/ondrej/repos/lpython/src/lpython/parser/parser.cpp", line 126
    Result<LPython::AST::Module_t*> res = parse(al, input, prev_loc, diagnostics);
  File "/Users/ondrej/repos/lpython/src/lpython/parser/parser.cpp", line 22
    p.parse(s, prev_loc);
  File "/Users/ondrej/repos/lpython/src/lpython/parser/parser.cpp", line 53
    m_tokenizer.set_string(inp, prev_loc);
  File "parser.yy", line 1107
  File "/Users/ondrej/repos/lpython/src/lpython/parser/semantics.h", line 879
    throw LCompilers::LCompilersException("The string is not recognized by the parser.");
LCompilersException: The string is not recognized by the parser.
```
and in CPython 3.10.2 we get:
```console
  File "/Users/ondrej/repos/lpython/b.py", line 3
    print "Hello", "World" 
    ^^^^^^^^^^^^^^^^^^^^^^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(...)?
```
So the above error is a good one. 

We could later on improve the parser to even parse the old syntax, but it's probably not worth it, as long as it gives a nice error message now.